app.js

```diff
@@ -1,148 +1,106 @@
-//Variables para llevar el registro del puntaje y el resultado general
+//Vars to register score and general result
 
 let score = 0;
 let totalQuestions = 5;
-
-// Función para manejar el evento submit del PRIMER formulario
-function handleQuestion1Submit(event) {
+let currentQuestion = 1;
+
+//Functions to handle submit form submission
+function handleQuestionSubmit(
+  event,
+  correctAnswer,
+  correctMessage,
+  incorrectMessage
+) {
   event.preventDefault();
 
   const selectedValue = document.querySelector(
-    'input[name="questionOne"]:checked'
+    `input[name="question${currentQuestion}"]:checked`
```

Comment:
You can put this string inside a variable, It will be easier to understand whats your goal. 

```
const checkedRadioButtonQuery = `input[name="question${currentQuestion}"]:checked`;
const selectedValue = document.querySelector(checkedRadioButtonQuery );
```

Reply:
I put the string inside a variable as suggested. Thank you.